packages/components/Notification/src/Notification.helper.tsx

```diff
@@ -1,6 +1,42 @@
 import React from 'react';
+import { ButtonProps, ButtonTokens, ButtonV1 as Button } from '@fluentui-react-native/button';
+import { mergeProps, stagedComponent } from '@fluentui-react-native/framework';
 import { Path } from 'react-native-svg';
 
+export type ActionButtonColorStates = { disabledColor; hoveredColor; pressedColor; focusedColor };
+type ActionButtonProps = ButtonProps & ButtonTokens & ActionButtonColorStates;
+export const ActionButton = stagedComponent((props: ActionButtonProps) => {
```

Comment:
nit, newlines between the declarations. I would also leave a simple 1-2 line comment explaining why ActionButton exists. Something like:


```
// We need to customize Notifiations' `action` slots' tokens based on Notification's variant prop. Compose doesn't let us easily do that via styling settings, so we introduce this helper component to customize tokens via props.